docs/getting-started/getting-started.adoc

```diff
@@ -8,13 +8,43 @@ You will need the following installed on your system
 
  * Android SDK or Android Studio from https://developer.android.com/studio/index.html[Google]
 
-It is reccomended you have the following installed  
+It is recommended you have the following installed
 
  * The latest version of http://openjdk.java.net/install/index.html[OpenJDK]
 
-== Installing
+== Using a Published SDK
 
-As of the writing of this guide, the Aerogear Android SDK is not in jcenter and needs to be installed manually.
+=== jcenter()
+
+To use a published version of the SDK you have to add jcenter to the build.grade file in your project's root directory
+
+[source, groovy]
+----
+allprojects {
+    repositories {
+        jcenter() // <-- Add This line
+        google()
+    }
+}
+----
+
+=== Reference the Dependencies
+
+In the build.gradle file of your app you can begin adding dependencies provided by the sdk.
```

Comment:
Tiny thing.

I think we should try to keep our capitalization of `sdk` consistent. So let's go with `SDK` here and blow.